style.css

```diff
@@ -0,0 +1,1067 @@
+}
+
+.nav-container {
+  display: flex;
+  align-items: flex-start;
+  text-transform: capitalize;
+  padding-top: 5px;
+  padding-bottom: 25px;
+  padding-left: 15px;
+  padding-right: 15px;
+  margin: 0;
+  background: #808080;
+}
+
+.nav-container img {
+  width: 160px;
+  height: 80px;
+  margin: 0;
+}
+
+.nav-btns {
+  display: flex;
+  margin-top: 12px;
+  margin-left: 15px;
+}
+
+.nav-item {
+  display: block;
+  list-style: none;
+  font-size: 18px;
+  font-weight: 550;
+  color: #fff;
+  cursor: pointer;
+}
+
+.nav-container a {
+  text-decoration: none;
+  border-radius: 10px;
+  padding: 10px 12px;
+}
+
+.cat-text a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.one-article a {
+  color: black;
+}
+
+.art-author a {
+  color: gray;
+  font-weight: 700;
+  font-size: 15px;
+}
+
+.one-article-2 a {
+  color: black;
+}
+
+.members-right a,
+.members-right p {
+  color: white;
+}
+
+.ship-p2 a {
+  color: white;
+}
+
+.per-flex a,
+.data a {
+  color: #006fc6;
+  filter: grayscale(1);
+}
+
+.art-para a {
+  color: #006fc6;
+  filter: grayscale(1);
+}
+
+.foot-para a {
+  color: #fff;
+}
+
+.last-links a {
+  color: #fff;
+}
+
+.lots-ofLinks a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.nav-container a:hover {
+  cursor: pointer;
+}
+
+.spacing-nav {
+  display: block;
+  margin-top: 8px;
+  margin-right: 5px;
+}
+
+.spacing-nav:hover {
+  background: #707070;
+}
+
+.more-btn {
+  display: flex;
+  background: #686868;
+  border-radius: 10px;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  padding-left: 10px;
+  padding-right: 10px;
+  margin-left: 5px;
+  margin-right: 40px;
+  font-size: 18px;
+  font-weight: 550;
+  text-transform: capitalize;
+  border: solid 1px #525252;
+}
+
+.more-btn:hover {
+  background: #505050;
+}
+
+.more-w {
+  margin-top: 5px;
+  display: inline-block;
+}
+
+.more-btn img {
+  width: 30px;
+  height: 30px;
+  margin-left: 7px;
+  filter: grayscale(1);
+}
+
+.searchArti-form i {
+  position: absolute;
+  margin-top: 17px;
+  margin-left: 20px;
+  font-size: 22px;
+  color: gray;
+  padding: 0;
+}
+
+.searchArti-form input {
+  display: block;
+  width: 285px;
+  height: 53px;
+  border-radius: 10px;
+  padding-left: 50px;
+  font-weight: 600;
+  margin-top: 12px;
+  border: none;
+}
+
+::placeholder {
+  font-size: 18px;
+}
+
+.text-seriousBtn {
+  display: flex;
+  justify-content: flex-end;
+  align-items: baseline;
+  cursor: pointer;
+}
+
+.miss-articles {
+  text-align: center;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding: 0;
+  margin-right: 90px;
+  font-size: 35px;
+  line-height: 45px;
+  width: 500px;
+  position: relative;
+}
+
+.seriously-btn {
+  width: 250px;
+  height: 40px;
+  display: flex;
+  position: relative;
+  margin-right: 50px;
+}
+
+.serious-text {
+  max-height: 40px;
+  font-size: 17px;
+  font-weight: 500;
+  color: #fff;
+  border: solid 1px #fff;
+  border-radius: 10px;
+  padding-top: 10px;
+  padding-right: 10px;
+  padding-bottom: 10px;
+  padding-left: 10px;
```

Comment:
The properties padding-top, padding-bottom, padding-left, padding-right can be replaced by padding.